packages/pds/src/api/com/atproto/server/createAccount.ts

```diff
@@ -40,6 +40,30 @@ export default function (server: Server, ctx: AppContext) {
       const now = new Date().toISOString()
       const passwordScrypt = await scrypt.genSaltAndHash(password)
 
+      if (ctx.cfg.phoneVerification.required && ctx.twilio) {
+        if (!input.body.verificationPhone) {
+          throw new InvalidRequestError(
+            'Phone number verification is required on this server and none was provided.',
+            'InvalidPhoneVerification',
```

Comment:
We might consider adding this error type to the lexicon.

Reply:
yeah i played kinda fast and loose cuz i know this is gonna be temporary. but yup probably a good idea 👍